utils/auc/src/auc.c

```diff
@@ -1989,6 +1992,34 @@ int main(int args, char *argv[]) {
 
 	req_add_selected_packages(&reqbuf);
 
+	int          fd;
```

Comment:
Variables should be declared at the top of the function.

Reply:
done